can_messages.asciipb

```diff
@@ -124,18 +124,7 @@ msg {
   }
 }
 
-# IDs: 9-14 Reserved
-msg {
-  id: 15
```

Comment:
Why this removal? If you're not making any changes to the message, there's no point in removing and adding additional noise to the diff, only to re-add it later.

If anything, I'd expect this diff

```diff
diff --git i/can_messages.asciipb w/can_messages.asciipb
index 04559d2..1b3e6da 100644
--- i/can_messages.asciipb
+++ w/can_messages.asciipb
@@ -127,11 +127,12 @@ msg {
 # IDs: 9-14 Reserved
 msg {
   id: 15
-  source: DRIVER_CONTROLS
+  source: DRIVER_CONTROLS_CENTER_CONSOLE
   # target: MOTOR_INTERFACE
   msg_name: "motor controller reset"
   can_data {
-    empty {
+    u8 {
+      field_name_1: "motor controller"
     }
   }
 }
```